util/uvmdvgen/env_pkg.sv.tpl

```diff
@@ -10,7 +10,7 @@ package ${name}_env_pkg;
 % for agent in env_agents:
   import ${agent}_agent_pkg::*;
 % endfor
-  import dv_lib_pkg::*;
+% import dv_lib_pkg::*;
```

Comment:
A % here seems wrong. Would you please try running the script with all combinations:
-c, -r, -c -r.

Reply:
Yes, you're right, this % was unintended. I've ran with various combination, and one question popped up: if has_ral is disabled, do we want to parameterize the scoreboard and base_vseq with RAL_T?

Reply:
nope. update those templates as well just the way we did for the env_cfg tpl.

Reply:
Sure, updated those templates as well